float32 vel
float32 error
bool frontBlocked
bool leftBlocked
bool rightBlocked